src/index.ts

```diff
@@ -175,12 +180,14 @@ export default function OpentracingPlugin<InstanceContext extends SpanContext>({
               // idempotent method to add helpers to the first context available (which will be propagated by apollo)
               addContextHelpers(context);
 
-              const name = getFieldName(info);
+              let name = getFieldName(info);
```

Comment:
As the function is defaulted we can keep this as a const and remove line 189:

```suggestion
              const name = createCustomSpanName(getFieldName(info), info);
```